pom.xml

```diff
@@ -93,6 +93,7 @@
     <jts.version>1.17.0</jts.version>
     <jaiext.version>1.1.16</jaiext.version>
     <netcdf.version>4.6.15</netcdf.version>
+    <eclipse.emf.version>2.15.0</eclipse.emf.version>
```

Comment:
So this change does not actually change the version number, just adjust to reflect xsd dependency changing to group `org.eclipse.emf` eh?

Reply:
The version number and groupId for org.eclipse.xsd both changed.

I wanted to pull out this version as a property to make it easier to coordinate between GT and GS.